src/app/admin/AdminPage.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import React, { ReactElement } from "react";
+import UsersTable from "@/app/admin/usersTable/UsersTable";
+import { Accordion, AccordionDetails, AccordionSummary } from "@mui/material";
+import { ExpandMore } from "@mui/icons-material";
+import TableSurface from "@/components/Tables/TableSurface";
+import CreateUserForm from "@/app/admin/createUser/CreateUserForm";
+import { faUsers, faUserPlus, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+import styled from "styled-components";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { UserRow } from "@/app/admin/adminActions";
+
+const PanelIcon = styled(FontAwesomeIcon)`
+    margin-right: 0.5em;
+`;
+
+interface Props {
+    userData: UserRow[];
+}
+
+// TODO VFB-23 Add accessibility tests for the admin page
+
+const AdminPage: React.FC<Props> = (props) => {
+    const adminPanels: [string, IconDefinition, ReactElement][] = [
+        ["USERS TABLE", faUsers, <UsersTable key={1} userData={props.userData} />],
```

Comment:
is `key={1}` required, especially if you have a `key={index}` already on the TableSurface?

Reply:
I thought the same thing, but I believe the linter or something was complaining because these objects were in a list. I'll see if it changes after fixing the above